### ISSUE TITLE
x230 | BIOS settings reset when time autosync is enabled on windows 10

### ISSUE THREAD

**Issue by vvill0w** [merge/skulls#299]
if i disable time syncing on windows the settings work just fine
but if i enable it, all of BIOS settings reset, including crucial fn_ctrl_swap

i tried everything but it didnt help
and i cant return to original bios since my backups got deleted by mistake forever

**Comment by merge** [comment]
what exactly is "time syncing" on windows? I think of getting the time via ntp on linux, but that can't be it. Setting an RTC's clock *should* not change thinks either, but that we could test on linux at least...

**Comment by vvill0w** [comment]
> what exactly is "time syncing" on windows? I think of getting the time via ntp on linux, but that can't be it. Setting an RTC's clock _should_ not change thinks either, but that we could test on linux at least...

by time syncing i meant Date & Time settings where you can toggle the option to set time automatically, via NTP (time.windows.com)

sorry for using incorrect terms

**Comment by tlaurion** [comment]
You imply that cmos gets reset by setting time through ntp?

**Comment by vvill0w** [comment]
i guess so, if its responsible for setting the time
ive already ordered a cmos battery, hoping itll fix my problem

**Comment by vvill0w** [comment]
> You imply that cmos gets reset by setting time through ntp?

i guess so, if its responsible for setting the time
ive already ordered a cmos battery, hoping itll fix my problem

**Comment by tlaurion** [comment]
> > You imply that cmos gets reset by setting time through ntp?
> 
> i guess so, if its responsible for setting the time
> ive already ordered a cmos battery, hoping itll fix my problem

Must be that to be honest. If your computer doesn't hold time, then it doesn't hold other cmos settings.

**Comment by vvill0w** [comment]
alright then, ill update this issue in 3 days when cmos battery arrives

**Comment by vvill0w** [comment]
no, it aint cmos' fault, even with a new one installed it still resets when i synchronize my clock in the settings
